dev/integration_tests/new_gallery/analysis_options.yaml

```diff
@@ -0,0 +1,20 @@
+include: package:flutter_lints/flutter.yaml
```

Comment:
Fly-by comment: Can we delete this file to apply the normal flutter lint configuration to the gallery and enforce a unified coding style across this repo?

Reply:
`always_specify_types` is a big lift here.

Reply:
According to https://dart.dev/tools/linter-rules/always_specify_types there's an auto-fix available for that lint. Presumably, running that over the code base here should fix most (if not all) cases automatically?

Reply:
dart fix got some things, but as of now:

```
jonahwilliams-macbookpro3:new_gallery jonahwilliams$ dart fix --dry-run
Computing fixes in new_gallery (dry run)... 3.4s
Nothing to fix!
```

Reply:
I think that just means I can click on a call site in the IDE to have it insert the type?

Reply:
actually dart fix broke all the null checks 😢

Reply:
lints fixed